ztf-sentinel-service/generate_supervisor_conf.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import sys
+import yaml
+
+
+def generate_conf(service):
+    if service not in config["sentinel"]["supervisord"].keys():
+        raise ValueError(f"{service} service def not found in config['supervisord']")
+
+    sc = ""
+    for k in config["sentinel"]["supervisord"][service].keys():
+        sc += f"[{k}]\n"
+        for kk, vv in config["sentinel"]["supervisord"][service][k].items():
+            sc += f"{kk} = {vv}\n"
+
+    with open(f"/app/supervisord_{service}.conf", "w") as fsc:
+        fsc.write(sc)
+
+
+def watcher(args):
+    generate_conf("watcher")
+
+
+with open("/app/config.yaml", "r") as f:
+    config = yaml.load(f, Loader=yaml.FullLoader)
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
```

Comment:
It is these kinds of things that have been replaced with fire elsewhere, right?

Reply:
Ah, right, forgot about this one. This script migrated from an old code base and I forgot to change it upon @stefanv's comment. Fixed now.

Reply:
Has this change not been committed? I think I am still seeing the earlier version, and no diff.